### ISSUE TITLE
Crash when clicking on Oracle DB table with millions of rows

### ISSUE THREAD

**Issue by cblage** [TablePlus/TablePlus#3470]
### Please answer the questions below, it helps us to keep track of the issue. Please do not ignore it or your issue will be closed.

1. Which driver are you using and version of it (Ex: PostgreSQL 10.0):
OracleDB - 23ai

3. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81):
Version 6.2.0 (577)

4. The steps to reproduce this issue:
* Open TablePlus
* Connect to OracleDB
* Click on table on sidebar with millions of rows

The crash doesn't happen always, but it happens more often than not.

Noted: If the bug is related to data, please attach an example SQL data.

Crash data: [crashlog.txt](https://github.com/user-attachments/files/18137325/crashlog.txt)

**Comment by cblage** [comment]
I don't think it has to do with the number of rows in the table since it's also crashing with a local version of the table with a few dozen rows. I think it has to do with having JSON in CLOB columns.

**Comment by huyphams** [comment]
Thank you for reaching out. Can you try our lasted release and see how it goes? I think you already enable beta updates so just go check for update for the build 595

**Comment by huyphams** [comment]
Hi, this hotpatch should work for JSON type
https://www.dropbox.com/scl/fi/hvu1ox8su6e98m95ihixz/TablePlus.dmg?rlkey=k4o6iswl42z8m8jtyer7onbxp&dl=0
it can decode the Oracle new JSON type! it will be included in the next update!
